docs/configuration.md

```diff
@@ -0,0 +1,121 @@
+# A Rough Guide to SCITT Configuration
+
+When SCITT-CCF nodes are first deployed, they are booted with initial [node configuration](https://microsoft.github.io/CCF/main/operations/configuration.html).
+
+Members registered in the inital node configuration must then be [activated](https://microsoft.github.io/CCF/main/governance/adding_member.html#activating-a-new-member).
+
+Members can then make and vote on [proposals](https://microsoft.github.io/CCF/main/governance/proposals.html) to update SCITT service configuration.
+
+Once SCITT is apropriately configured members can vote to [open](https://microsoft.github.io/CCF/main/governance/open_network.html#opening-the-network) the service.
+- Note: SCITT does not require CCF style `Users` to be configured.
+
+# SCITT Configuration
```

Comment:
Should be `##` as there typically is only a single top-level heading per document.